e2e/tests/hmctsSuperUserAdministersCase_test.js

```diff
@@ -0,0 +1,29 @@
+const config = require('../config.js');
+
+const orderCaseData = require('../fixtures/testData/caseDataWithOrderCollection.json');
+const orderFunctions = require('../helpers/generated_order_helper');
+const blankOrder = require('../fixtures/orders/blankOrder.js');
+
+Feature('Case administration by super user');
+
+let caseId;
+
+BeforeSuite(async I => caseId = await I.submitNewCaseWithData(orderCaseData));
+
+Before(async I => await I.navigateToCaseDetailsAs(config.hmctsSuperUser, caseId));
+
+Scenario('HMCTS super user removes an order from a case', async (I, caseViewPage, loginPage, removeOrderEventPage) => {
+  await caseViewPage.goToNewActions(config.superUserActions.removeOrder);
+  let order = orderCaseData.caseData.orderCollection[0];
+  const labelToSelect = order.value.title + ' - ' + order.value.dateOfIssue;
+  removeOrderEventPage.selectOrderToRemove(labelToSelect);
+  await I.retryUntilExists(() => I.click('Continue'), removeOrderEventPage.fields.reason);
+  removeOrderEventPage.addRemoveOrderReason('Entered incorrect order');
+  await I.completeEvent('Submit');
+  I.seeEventSubmissionConfirmation(config.superUserActions.removeOrder);
+  const issuedDate = new Date(2020, 4,26, 14, 33);
+  order = {...blankOrder, ...{
+    document: 'Blank order (C21).pdf',
+  } };
+  await orderFunctions.assertOrder(I, caseViewPage, order, issuedDate, false, true);
```

Comment:
I think it's hard to read - it would be much better if method names says what we are asserting. Impossible to say now what is happening without going deeper, test should be a bit cleaner.

Reply:
I tried to reuse existing assertion which asserts order, now method asserts for removed order too which is exactly same thing but with different title. It made sense to me but happy to take suggestion and work on it as a small follow up PR.

Reply:
Please do, false / true seems as magic values - when reading, no clue what's really being asserted. Such helpers should have more readable variations:
assertFinalOrder
assertOrderHidden

(something along the lines)